manager/Makefile

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+include ../include.mk
+
 .PHONY : all build test push container clean .project
```

Comment:
This "test" PHONY target is no longer necessary as it's defined near the target below.